### DISCUSSION TITLE
vscode theme?

### DISCUSSION THREAD

**Discussion by yassinosman** [microsoft/vscode-discussions#490] [Extension Development QnA] [answered]
Hi everyone

Does anyone know the name of this vscode theme?

![image](https://user-images.githubusercontent.com/45969373/221130520-5e9a6328-67b5-409a-b9c3-de768e0db58c.png)

thanks

**Comment by tyler-holm** [comment, marked as answer]
Looks a lot like Monokai Pro with bracket pair colorization turned on.

You can check it out here: https://vscodethemes.com/e/monokai.theme-monokai-pro-vscode/monokai-classic?language=javascript
